src/Parser/ProtectionWarrior/Modules/Core/HealingDone.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import { formatThousands, formatNumber } from 'common/format';
+
+import CoreHealingDone from 'Parser/Core/Modules/HealingDone';
+
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+class HealingDone extends CoreHealingDone {
```

Comment:
Recent changes made it so this can be replaced with 2 lines in your CombatLogParser to get a version including a healing vs overhealing bar; https://github.com/MartijnHols/WoWAnalyzer/blob/f90370b34593f84fa0d3d90a92a551c9c6e7660d/src/Parser/HolyPaladin/CombatLogParser.js#L67